packages/contracts/src/interfaces/IERC20Token.sol

```diff
@@ -8,10 +8,11 @@ import "@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol";
 
 /**
  * @notice Extended ERC-20 token interface used internally in OpenGSN modules.
- * Renamed to avoid conflict with OZ namespace. Includes IERC20, ERC20Metadata and 'mint(uint256)'.
- * @notice Interface of the ERC20 standard as defined in the EIP.
+ * @dev Renamed to avoid conflict with OZ namespace. Includes IERC20, ERC20Metadata.
```

Comment:
Use 'notice' - dev is not presented in the generated .md files. Our .md files are for devs anyway.